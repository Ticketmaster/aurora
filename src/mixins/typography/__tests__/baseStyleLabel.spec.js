import BASE_STYLE_LABEL from "../baseStyleLabel";

describe("BASE_STYLE_LABEL", () => {
  it("should return the correct values", () => {
    expect(BASE_STYLE_LABEL).toMatchSnapshot();
  });
});
