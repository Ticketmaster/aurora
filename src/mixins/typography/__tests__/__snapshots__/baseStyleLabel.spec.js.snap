// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BASE_STYLE_LABEL should return the correct values 1`] = `
Array [
  "
  ",
  "
  color: ",
  [Function],
  ";
",
  ";
  ",
  "
  ",
  "font-weight: 600",
  ";
",
  ";
  ",
  "
  line-height: ",
  "1.25",
  ";
",
  ";
",
]
`;
